OpenEdXMobile/src/main/java/org/edx/mobile/util/VideoUtil.java

```diff
@@ -189,4 +193,28 @@ public static boolean isYoutubeAPISupported(Context context) {
             return false;
         }
     }
+
+    /**
+     * Method to check if the EncodedVideo is a Native Video or Not
+     * A unit can have zero, single or multiple EncodedVideos
+     * If Youtube have the stream priority but native video is also download in the app then
+     * the unit will be treated as native video instead of youtube.
+     *
+     * @return True if given unit is a native video unit or downloaded video is available
+     */
```

Comment:
/**
     * Method to check if the EncodedVideo is a Native Video or Not
     * A unit can have zero, single, or multiple EncodedVideos
     * A downloaded video is prioritized over a recommended Youtube video
     *
     * @return True if the given unit is a native video unit or downloaded video is available
     */